### DISCUSSION TITLE
Collapse sidebar options

### DISCUSSION THREAD

**Discussion by omen124** [Able-Bear-Software/tab-nodes-tree#66] [Ideas / Feature Request]
When you have the sidebar open, there are a few improvements that I would love to see:

1. The ability to adjust the with of the bar
2. The ability to change the font and spacing settings, in case you may have difficulty reading or want it in a more minimalist style
3. The ability to get the bar to collapse and only show the favicons, of collapse completely like the autohide function of the taskbar in Windows)
4. The ability for the bar to auto-hide (in conjunction with the point above) you can mouse over the icons to get them to expand to see the full names
5. When considering an expanding bar, the web page content should not move and the bar should slide over the top of it

**Comment by zeroarst** [comment]
For points 1, 3, 4, and 5, please refer to [this Q&A](https://www.tab-nodes-tree.app/qna?q=change-side-panel-size). Due to certain limitations, these features cannot be implemented.
As for point 2, I can add the option to change font and icon sizes. Could you provide more details on the spacing adjustments you're looking for?
